NewForce-Cohort-6/tabloid-cli-coven-of-the-lunar-owls
# Ticket #17 - Remove Post

# Description

This PR is for [Ticket #17](https://trello.com/c/gaZsGkHu), which should allow the user to navigate to Post Management from the Main Menu and then remove a post from the list of posts using option 5 of the Post Menu. This has been tested in the CLI and displays the expected result from the database, which is the removal of a post from the list upon the user's request. 

# Testing Instructions

1. git fetch --all
2. git checkout ac-deletePosts to switch to my branch.
If you don't see the most recent version of the files, try git pull origin ac-deletePosts.
3. Execute TabloidCLI from inside of Visual Studio
4. Select option 4, Post Management, from Main Menu.
5. Select option 5, Remove Post, from Post Menu.
6. Select a post to remove from the list of posts. 
7. Select option 1, List Posts, from the Post Management menu and view your changes. 

# Checklist:
